### DISCUSSION TITLE
pytorch: command not found

### DISCUSSION THREAD

**Discussion by kylanj7** [mrdbourke/pytorch-deep-learning#977] [Q&A]
after setting up :~/Desktop/ztm-pytorch-course$ and installing pytorch 2.3.1, conda, and cuda 12.5.0 I am recieving this error. I tried installing the cuda toolkit inside of /Desktop/ztm-pytorch-course/env twice now and the error is still returning. Any advice?

after checking the env with this script, I get a tensor output but my GPU comes back as false. 

import pandas as pd
import numpy as np
import torch
import sklearn
import matplotlib
import torchinfo, torchmetrics

# Check PyTorch access (should print out a tensor)
print(torch.randn(3, 3))

# Check for GPU (should return True)
print(torch.cuda.is_available())

**Comment by aldragan0** [comment]
So you mention that you've installed cuda toolkit but your gpu is not recognized in python.
Have you checked that you have the latest nvidia driver installed? If you open a terminal and type `nvidia-smi` what do you get?

Also, have you tried installing it the recommended way from the [pytorch website](https://pytorch.org/get-started/locally/)?
